server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/sql/TestSqlHostVariable.java

```diff
@@ -350,4 +360,9 @@ void testBinaryHostVariableArray3() {
   void testBinaryHostVariableArray4() {
     UseCaseEngine.runTest(BINARY_ARR_TEXT4, ImmutableList.of(), ImmutableMap.of());
   }
+
+  @Test
```

Comment:
Add test cases validating
- range being 2-48 
- occurrence number being positive number <32K

Reply:
Resolved in latest commit.